gallery/cassandra-mi-insights/microsoft.documentdb-cassandraclusters.json

```diff
@@ -0,0 +1,17 @@
+{
+	"$schema": "https://raw.githubusercontent.com/microsoft/Application-Insights-Workbooks/master/schema/gallery.json",
+	"version": "TemplateGallery/1.0",
+	"categories": [
+		{
+			"id": "CassandraMI",
+			"name": "Cassandra Managed Instance",
+			"templates": [
+				{
+					"id": "Workbooks/Azure Managed Instance for Apache Cassandra/Cassandra MI Insights",
+					"author": "Microsoft",
+					"name": "Azure Managed Instance for Apache Cassandra Insights"
```

Comment:
You should also add a description here, and for the insights type, can we call it `amicassandra-insights`?